src/Dispatcher.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Ntfy;
+
+use stdClass;
+use Ntfy\Exception\NtfyException;
+
+class Dispatcher
```

Comment:
Could also rename it to `Ntfy` if you think that's more fitting to the way you had the library before.

Reply:
Could also be `Client` as it's the one sending the message :)